ironfish/src/workerPool/job.test.slow.ts

```diff
@@ -36,5 +36,28 @@ describe('Worker Pool', () => {
 
     expect(result.valid).toBe(true)
     expect(workerPool.completed).toBe(1)
-  }, 60000)
+  })
+
+  it('boxMessage and unboxMessage', async () => {
```

Comment:
Sorry, did a rebase but accidentally added this test back in 😬

Reply:
no worries, fixed